lib/ansible/modules/network/cloudengine/ce_lacp.py

```diff
@@ -0,0 +1,548 @@
+        required: false
+        type: int
+    collector_delay:
+        description:
+            - Value of delay time in units of 10 microseconds.
+        required: false
+        type: int
+    max_active_linknumber:
+        description:
+            - Max active linknumber in link aggregation group.
+        required: false
+        type: int
+    select:
+        description:
+            - Select priority or speed to preempt.
+        required: false
+        choices: ['Speed', 'Prority']
+        type: str
+    member_if:
+        description:
+            - The member interface of eth-trunk that is selected is merge priority.
+        required: false
+        type: str
+    priority:
+        description:
+            - The priority of eth-trunk member interface,and 'member_if' is need when priority is not none.
+        required: false
+        type: int
+    global_priority:
+        description:
+            - Configure lacp priority on system-view.
+        required: false
+        type: int
+    state:
+        description:
+            - Manage the state of the resource.
+        required: false
+        default: present
+        choices: ['present','absent']
+        type: str
+    provider:
+        type: dict
+        description:
+            - A dict specified some options about connection  that is not required from version 2.9.
+            - And provider is unnecessary when using network_cli and will be ignored.
+        required: false
+'''
+EXAMPLES = '''
+- name: eth_trunk module test
+  hosts: cloudengine
+  connection: local
+  gather_facts: no
+
+  tasks:
+  - name: Ensure Eth-Trunk100 is created, add two members, and set to mode lacp-static
+    ce_eth_trunk:
+      trunk_id: 100
+      members: ['10GE1/0/24','10GE1/0/25']
+      mode: 'lacp-static'
+      state: present
+'''
+
+RETURN = '''
+proposed:
+    description: k/v pairs of parameters passed into module
+    returned: always
+    type: dict
+    sample: {"trunk_id": "100", "members": ['10GE1/0/24','10GE1/0/25'], "mode": "lacp-static"}
+existing:
+    description: k/v pairs of existing Eth-Trunk
+    returned: always
+    type: dict
+    sample: {"trunk_id": "100", "hash_type": "mac", "members_detail": [
+            {"memberIfName": "10GE1/0/25", "memberIfState": "Down"}],
+            "min_links": "1", "mode": "manual"}
+end_state:
+    description: k/v pairs of Eth-Trunk info after module execution
+    returned: always
+    type: dict
+    sample: {"trunk_id": "100", "hash_type": "mac", "members_detail": [
+            {"memberIfName": "10GE1/0/24", "memberIfState": "Down"},
+            {"memberIfName": "10GE1/0/25", "memberIfState": "Down"}],
+            "min_links": "1", "mode": "lacp-static"}
+updates:
+    description: command sent to the device
+    returned: always
+    type: list
+    sample: ["interface Eth-Trunk 100",
+             "mode lacp-static",
+             "interface 10GE1/0/25",
+             "eth-trunk 100"]
+changed:
```

Comment:
it is not needed to add this here because AFAIK all ansible modules return it always
```suggestion
changed:
```